data/tlg0341/tlg002/tlg0341.tlg002.perseus-grc1.xml

```diff
@@ -1,1606 +1,1604 @@
-<l n="1428">σκιὰ καλύψει πέρραν, ἀμβλύνων σέλας.</l>
-<l n="1429">Λοκρὸν δʼ ὁποῖα παῦρον ἀνθήσας ῥόδον,</l>
-<l n="1430">καὶ πάντα φλέξας, ὥστε κάγκανον στάχυν, </l>
-<l n="1431">αὖθις παλιμπλώτοιο γεύσεται φυγῆς,</l>
-<l n="1432">μόσσυνα φηγότευκτον, ὡς λυκοψίαν</l>
-<l n="1433">κόρη κνεφαίαν, ἄγχι παμφαλώμενος,</l>
-<l n="1434">χαλκηλάτῳ κνώδοντι δειματουμένη.</l>
-<l n="1435">πολλοὶ δʼ ἀγῶνες καὶ φόνοι μεταίχμιοι </l>
-<l n="1436">λύσουσιν ἀνδρῶν οἱ μὲν ἐν γαίᾳ <note n="1" place="unspecified" anchored="yes">One expects "on the sea," but no satisfactory emendation has been proposed.</note> πάλας</l>
-<l n="1437">δειναῖσιν ἀρχαῖς ἀμφιδηριωμένων,</l>
-<l n="1438">οἱ δʼ ἐν μεταφρένοισι βουστρόφοις χθονός,</l>
-<l n="1439">ἕως ἂν αἴθων εὐνάσῃ βαρὺν κλόνον,</l>
-<l n="1440">ἀπʼ Αἰακοῦ τε κἀπὸ Δαρδάνου γεγὼς </l>
-<pb id="p.614"/>
-<l n="1441">Θεσπρωτὸς ἄμφω καὶ Χαλαστραῖος λέων,</l>
-<l n="1442">πρηνῆ θʼ ὁμαίμων πάντα κυπώσας δόμον</l>
-<l n="1443">ἀναγκάσῃ πτήξαντας Ἀργείων πρόμους</l>
-<l n="1444">σῆναι Γαλάδρας τὸν στατηλάτην λύκον</l>
-<l n="1445">καὶ σκῆπτρʼ ὀρέξαι τῆς πάλαι μοναρχίας. </l>
-<l n="1446">ᾧ δὴ μεθʼ ἕκτην γένναν αὐθαίμων ἐμὸς</l>
-<l n="1447">εἷς τις παλαιστής, συμβαλὼν ἀλκὴν δορὸς</l>
-<l n="1448">πόντου τε καὶ γῆς κεἰς διαλλαγὰς μολών,</l>
-<l n="1449">πρέσβιστος ἐν φίλοισιν ὑμνηθήσεται,</l>
-<l n="1450">σκύλων ἀπαρχὰς τὰς δορικτήτους λαβών. </l>
-<l n="1451">τί μακρὰ τλήμων εἰς ἀνηκόους πέτρας,</l>
-<l n="1452">εἰς κῦμα κωφόν, εἰς νάπας δασπλήτιδας</l>
-<l n="1453">βαύζω, κενὸν ψἀλλουσα μάστακος κρότον;</l>
-<l n="1454">πίστιν γὰρ ἡμῶν Λεψιεὺς ἐνόσφισε,</l>
-<l n="1455">ψευδηγόροις φήμαισιν ἐγχρίσας ἔπη, </l>
-<l n="1456">καὶ θεσφάτων πρόμαντιν ἀψευδῆ φρόνιν,</l>
-<l n="1457">λέκτρων στερηθεὶς ὧν ἐκάλχαινεν τυχεῖν.</l>
-<l n="1458">θήσει δʼ ἀληθῆ. σὺν κακῷ δέ τις μαθών,</l>
-<l n="1459">ὅτʼ οὐδὲν ἔσται μῆχος ὠφελεῖν πάτραν,</l>
-<l n="1460">τὴν φοιβόληπτον αἰνέσει χελιδόνα. </l>
-<l n="1461">Τόσσʼ ἠγόρευε, καὶ παλίσσυτος ποσὶν</l>
-<l n="1462">ἔβαινεν εἱρκτῆς ἐντός. ἐν δὲ καρδίᾳ</l>
-<l n="1463">σειρῆνος ἐστέναξε λοίσθιον μέλος,</l>
-<l n="1464">Κλάρου Μιμαλλών, ἢ Μελαγκραίρας κόπις</l>
-<l n="1465">Νησοῦς θυγατρός, ἤ τι Φίκιον τέρας, </l>
-<l n="1466">ἑλικτὰ κωτίλλουσα δυσφράστως ἔπη.</l>
-<l n="1467">ἐγὼ δὲ λοξὸν ἦλθον ἀγγέλλων, ἄναξ,</l>
-<pb id="p.616"/>
-<l n="1468">σοὶ τόνδε μῦθον παρθένου φοιβαστρίας,</l>
-<l n="1469">ἐπεί μʼ ἔταξας φύλακα λαΐνου στέγης</l>
-<l n="1470">καὶ πάντα φράζειν κἀναπεμπάζειν λόγον </l>
-<l n="1471">ἐτητύμως ἄψορρον ὤτρυνας τρόχιν.</l>
-<l n="1472">δαίμων δέ φήμας εἰς τὸ λῷον ἐκδραμεῖν</l>
-<l n="1473">τεύξειεν, ὅσπερ σῶν προκήδεται θρόνων,</l>
-<l n="1474">σώζων παλαιὰν Βεβρύκων παγκληρίαν.</l>
-</div1>
-</body>
-</text>
-</TEI.2>
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/9.1/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xmlns:tei="http://www.tei-c.org/ns/1.0">
```

Comment:
only `<TEI xmlns="http://www.tei-c.org/ns/1.0">` is needed. (And lang code is incorrect here).